src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -217,6 +223,90 @@ def create_data_set(self, dataset_name, options = {}):
         response_json = self.request_handler.perform_request("POST", custom_args, expected_code = [201])
         return response_json
 
+    def create_default_data_set(self, dataset_name, default_type):
+        """
+        Create a dataset with default options set.
+        Default options depend on the requested type.
+
+        Parameters
+        ----------
+            dataset_name
+            default_type: "partitioned" or "sequential" or "classic" or "c" or "binary" or "vsam"
+
+        Returns
+        -------
+        json
+            A JSON containing the result of the operation
+        """
+
+        if default_type not in zos_file_constants["SupportedDefaultDataSets"]:
+            raise exceptions.UnsupportedDefaultDataSetRequested
+
+        options = {}
+
+        if default_type == "partitioned":
+            options = {
+                "alcunit": "CYL",
+                "dsorg": "PO",
+                "primary": 1,
+                "dirblk": 5,
+                "recfm": "FB",
+                "blksize": 6160,
+                "lrecl": 80
+            },
+        elif default_type == "sequential":
+            options = {
+                "alcunit": "CYL",
+                "dsorg": "PS",
+                "primary": 1,
+                "recfm": "FB",
+                "blksize": 6160,
+                "lrecl": 80
+            },
+        elif default_type == "classic":
+            options = {
+                "alcunit": "CYL",
+                "dsorg": "PO",
+                "primary": 1,
+                "recfm": "FB",
+                "blksize": 6160,
+                "lrecl": 80,
+                "dirblk": 25
+            }
+        elif default_type == "c":
+            options = {
+                "dsorg": "PO",
+                "alcunit": "CYL",
+                "primary": 1,
+                "recfm": "VB",
+                "blksize": 32760,
+                "lrecl": 260,
+                "dirblk": 25
+            }
+        elif default_type == "binary":
+            options = {
+                "dsorg": "PO",
+                "alcunit": "CYL",
+                "primary": 10,
+                "recfm": "U",
+                "blksize": 27998,
+                "lrecl": 27998,
+                "dirblk": 25
+            }
+        elif default_type == "vsam":
+            options = {
+                "dsorg": "INDEXED",
+                "alcunit": "KB",
+                "primary": 840
+            }
```

Comment:
VSAM is a special data set type that cannot be created with the same API as other types.

To create a VSAM, you need to use the "Access Method Services Interface" documented [here](https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-access-method-services). This is added scope outside of the issue addressed in this PR, so I'd suggest removing VSAM for now and we can handle it as a separate issue in the future 🙂

Reply:
Alright, thank you, I'll edit this!